flight/Libraries/uavtalk.c

```diff
@@ -254,6 +261,26 @@ UAVTalkRxState UAVTalkProcessInputStreamQuiet(UAVTalkConnection connectionHandle
 		if (iproc->rxCount < 4)
 			break;
 
+		if (iproc->type == UAVTALK_TYPE_FILEREQ) {
```

Comment:
Do we want support for receiving files on the flight side? Could be useful for PicoC replacement if it ever eventuates. Might be safer to handle it here and error out in any case.